### ISSUE TITLE
[Request] Add back 'items-registry-name-only' category

### ISSUE THREAD

**Issue by Seeloewen** [maruohon/tellme#39]
In versions past TellMe for 1.17 the option to use 'items-registry-name-only' to dump the items with only their registry ID is no longer available. I would find it very useful if this could be added back. Is there anything holding it back from being readded?

**Comment by maruohon** [comment]
It should still be there, although it looks like in the recent Fabric versions it's now called just `items-registry-name`. But the Forge versions still use the name `items-registry-name-only`.

**Comment by Seeloewen** [comment]
Alright, thank you, that resolves my issue! Still it's a bit confusing, no? Would there be a possibility to unify the command across the two versions?

**Comment by maruohon** [comment]
Yeah it is rather annoying that the Fabric and Forge versions have diverged somewhat. There are a small number of small features in one or the other that might not be in the other version. Plus many of the dump names are different, in part because some of them are named after the names that the mappings call them, and the MCP or Mojmap names used in Forge are often different from the Yarn names that I use for Fabric projects.

An additional problem is that the tab completion doesn't work for the dump names atm... I should really fix that too at some point.
